README.rst

```diff
@@ -34,8 +35,72 @@ To install from source:
 
 .. installation-end-marker
 
-Next Steps
-----------
+Getting Started
+===============
+
+Sign up for the Leap quantum cloud service here:
+`Leap signup <https://cloud.dwavesys.com/leap/signup>`_. Leap provides a
+browser-based integrated development environment (IDE) with Ocean preinstalled.
+
+Start learning with the following D-Wave resources:
+
+* `System Documentation <https://docs.dwavesys.com/docs/latest/index.html>`_ to
+  learn about quantum computers and how to use them.
+
+* `Getting Started with Ocean <https://docs.ocean.dwavesys.com/en/stable/getting_started>`_
+  to install and start coding with Ocean software.
+
+* `dwave-examples <https://github.com/dwave-examples>`_ for code examples
+  and Jupyter Notebooks.
+
+* `D-Wave website <https://www.dwavesys.com/learn/resource-library>`_ for
+  whitepapers and additional resources.
+
+Example Quantum Program
+-----------------------
+
+The following lines of code solve and visualize a
+`random <https://docs.ocean.dwavesys.com/en/stable/docs_dimod/reference/generators.html>`_
+problem on a quantum computer.
+
+>>> from dwave.system import DWaveSampler, EmbeddingComposite
+>>> from dimod.generators import ran_r
+>>> import dwave.inspector
```

Comment:
ok, I'm going to be ultra nitpicky here, I think the order should be
```python
>>> import dwave.inspector
>>> from dimod.generators import ran_r
>>> from dwave.system import DWaveSampler, EmbeddingComposite
```
`from` after `import` and then alphabetical.